src/Files.App/Data/Contexts/Multitasking/MultitaskingContext.cs

```diff
@@ -48,13 +48,13 @@ private void AppModel_PropertyChanged(object? sender, PropertyChangedEventArgs e
 		{
 			UpdateCurrentTabIndex();
 		}
-		private void BaseMultitaskingControl_OnLoaded(object? sender, IMultitaskingControl control)
+		private void BaseMultitaskingControl_OnLoaded(object? sender, ITabView control)
```

Comment:
It might be better to rename the method to `BaseTabView_OnLoaded`

Reply:
Kinda out of scope